### PR TITLE
add recipe for segregation

### PR DESCRIPTION
**Pull request by renanxcortes**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from an official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@knaaptime and @sjsrey are also maintainers of this package and are listed in the `recipe-maintainers`.

Thank you!

### CODE REVIEW

**Comment by sjsrey** [comment]
I am willing to be listed as a maintainer.

**Comment by knaaptime** [comment]
I'm happy to help maintain

**Comment by knaaptime** [comment]
`warnings` shouldn't be in there because it's part of the standard library

**Comment by renanxcortes** [comment]
> `warnings` shouldn't be in there because it's part of the standard library

Yes, I thought about that also... but, I've been trying to figure what might be wrong with the PR...
In the Travis log it says that
`Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml`

**Comment by knaaptime** [comment]
hm. guess we'll have to keep debugging. We will need to remove warnings though, because its not a conda package so it'll crash the install

from the most recent built:
```
conda.exceptions.ResolvePackageNotFound: 
  - warnings
```

**Comment by knaaptime** [comment]
ah, i found it. its trying to download pytest-runner which is what's causing the failure. 


```
 "Be sure to add all dependencies in the meta.yaml  url=%s" % url)
    RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.org/simple/pytest-runner/
```

need to remove [this](https://github.com/pysal/segregation/blob/3a298c15909278b1794a6cd630a59de1fd0c79b1/setup.py#L56) line in setup.py

**Comment by renanxcortes** [comment]
> ah, i found it. its trying to download pytest-runner which is what's causing the failure.
> 
> ```
>  "Be sure to add all dependencies in the meta.yaml  url=%s" % url)
>     RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.org/simple/pytest-runner/
> ```
> 
> need to remove [this](https://github.com/pysal/segregation/blob/3a298c15909278b1794a6cd630a59de1fd0c79b1/setup.py#L56) line in setup.py

Nice! Thank you! I was trying to change something in the yaml! I'll change this line in the setup, release again and see how it goes!

**Comment by knaaptime** [comment]
🎉

**Comment by renanxcortes** [comment]
Thanks @knaaptime , I think now it is ready to go! 🎉🎉🎉

File: recipes/segregation/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (chrisburr) [inline recipes/segregation/meta.yaml]: Can you remove these comments from the example

File: recipes/segregation/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  description: |
+    The PySAL segregation module allow users to estimate several segregation measures and perform 
+    inference for single measures and comparative inference in a concise way.
+  doc_url: http://segregation.readthedocs.io/
+  dev_url: https://github.com/pysal/segregation
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (chrisburr) [inline recipes/segregation/meta.yaml]: Can you remove this comment from the example

File: recipes/segregation/meta.yaml (hunk 1 above)
Comment (renanxcortes) [reply recipes/segregation/meta.yaml]: Of course, just solved this in the last commit! Please let me know if anything else is needed!

File: recipes/segregation/meta.yaml (hunk 2 above)
Comment (renanxcortes) [reply recipes/segregation/meta.yaml]: Of course, just solved this in the last commit! Please let me know if anything else is needed!

**Comment by renanxcortes** [comment]
Thank you for merging it!
Best,
Renan

**Pull request merged**